### PR TITLE
und-54, 55

### PR DESCRIPTION
**Pull request by Ashish-srij**
Header Mega menu and mobile menu

### CODE REVIEW

File: stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader.scss [hunk 1]
```diff
@@ -70,8 +79,9 @@
 
     .middle-logo {
       @extend %img_none;
-
-      margin: 0 $spacing-80;
+      position: fixed;
+      left: 50%;
+      transform: translateX(-50%);
```
Comment (shishirphadke) [inline stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader.scss:84]: @Ashish-srij kindly use `@include transform(translateX(-50%));` mixin here

File: stories/Atom/Navigation/Menu-items/Menu-Items.scss [hunk 2]
```diff
@@ -3,7 +3,7 @@
 @import '../../../assets/scss/variables';
 
 // * menu item row start *//
-.menu_item {
+.menu-item a {
```
[the remark this answers is not in the document]
Comment (Ashish-srij) [reply stories/Atom/Navigation/Menu-items/Menu-Items.scss]: Done.

File: stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader.scss (hunk 1 above)
Comment (Ashish-srij) [reply stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader.scss]: Done.

File: stories/Components/Navigationcomponents/Mainnavigation/MegaMenu/MegaMenu.jsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Image } from '../../../../Atom/Images/Image/Image';
+import './MegaMenu.scss';
+
+const MegaMenu = ({ text }) => (
+  <div className="cell large-12 mega-nav-option" menu-item-id="headerlink2">
+    <div className="grid-x">
+      <div className="cell large-3">
+        <ul className="submenu">
+          <li id="submenu1-headerlink1" className="active"><a href="#" className="sub-link">Page Title</a></li>
```
[the remark this answers is not in the document]
Comment (Ashish-srij) [reply stories/Components/Navigationcomponents/Mainnavigation/MegaMenu/MegaMenu.jsx:10]: @shishirphadke We are using Id to get data for particular "li" for mega menu's right side part. We can't remove the ID or if it is necessary to remove it then how can we manage the right side data. Please advice.

File: stories/Components/Navigationcomponents/Mainnavigation/MobileNav/MobileNav.jsx [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+  langSelect,
+}) => {
+  useEffect(() => {
+    init(locale);
+  }, [locale]);
+
+  return (
+    <div className="cell large-12 mobile-nav">
+      <div className="grid-x">
+        <div className="cell small-12 mob-links">
```
[the remark this answers is not in the document]
Comment (Ashish-srij) [reply stories/Components/Navigationcomponents/Mainnavigation/MobileNav/MobileNav.jsx:25]: @shishirphadke We have a separate cell for mobile nav. From my side, it is showing fine for structure. Please advice.

File: stories/assets/js/navigation.js [hunk 5]
```diff
@@ -1,148 +1,77 @@
...
 
-  megaNavOption.mouseleave(() => {
-    megaNavOption.removeClass('show-mega');
-    megaNavOption.removeAttr('menu-item-id');
+  $('.mega-wrapper').on('mouseenter', '.submenu li', function () {
+    $(this).addClass('active');
+    $(this).siblings().removeClass('active');
+    const id = $(this).attr('id');
+    $(this).parents('.mega-wrapper').find(`[data-id='${id}']`).addClass('active-content');
+    $(this).parents('.mega-wrapper').find(`[data-id='${id}']`).siblings()
```
[the remark this answers is not in the document]
Comment (Ashish-srij) [reply stories/assets/js/navigation.js:41]: Done

File: stories/assets/js/navigation.js [hunk 6]
```diff
@@ -1,148 +1,77 @@
...
-    megaNavOption.removeAttr('menu-item-id');
+  $('.mega-wrapper').mouseleave(function () {
+    $(this).find('.mega-nav-option.show-mega').removeClass('show-mega');
   });
 
   midNav.mouseleave(() => {
-    if (megaNavOption.hasClass('show-mega')) {
-      megaNavOption.removeClass('show-mega');
-    }
+    // $('.mega-nav-option').removeClass('show-mega');
```
[the remark this answers is not in the document]
Comment (Ashish-srij) [reply stories/assets/js/navigation.js:29]: Done

**Pull request merged**